content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,267 @@
+# dvc.api.params_show()
+
+Load <abbr>parameters</abbr> (name and values) tracked in a <abbr>DVC
+project</abbr>.
+
+```py
+def params_show(
+    *targets: str,  # Optional
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+## Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves <abbr>params</abbr> keys and values from a <abbr>DVC project</abbr>
+and returns a JSON structure such as:
+
+```json
+{
+  "split": 0.2,
+  "seed": 20170428
+}
+```
+
+Without arguments, this function will retrieve all params from all tracked
+parameter files (used in any `dvc.yaml` file). This applies to the current
```

Comment:
Please confirm this is correct @daavoo , thanks.

Reply:
👍

Reply:
I think it will only retrieve params from `dvc.yaml` in the current directory, right? See https://github.com/iterative/dvc/issues/7926.